omnibus/config/projects/chef-server.rb

```diff
@@ -46,6 +46,7 @@
 dependency "openresty-lpeg"  # lua-based routing
 dependency "runit"
 dependency "chef_backup-gem" # chef-server-ctl backup
+dependency "rest-client-gem" # chef-server-ctl server-admins commands
```

Comment:
So...all the _acl endpoints are not written yet, which is why the server-admins commands need `rest-client`. Proposed version 2 of server-admins will add that endpoint in, update the CLI, and remove that dependency (much like the iterative development we did for key rotation). This is the best way to do it iteratively imo. For the whole plan, see here:

https://gist.github.com/tylercloke/a8d4bc1b915b958ac160#version-2

But yes, I can move it into `chef-server-ctl.rb` at the very least.